src/test/java/com/mathworks/ci/BuildArtifactActionTest.java

```diff
@@ -0,0 +1,233 @@
+package com.mathworks.ci;
```

Comment:
A number of changes happened to `RunMatlabBuildBuilder.java` but I don't see any test updates for those?

Reply:
Updated tests per new changes